schema_vdw.sql

```diff
@@ -1,115 +1,29 @@
+--@(#) script.ddl
+
+/*
+  For CODI at North Carolina	
+	Updated October 20, 2022 to conform with CODI DM 4.2
+	1. Based on a decision to not maintain population statistics in data owner CODI datamarts, 
+		the statistics fields are being removed from this table. However the VDW CENSUS_DEMOG 
+		table continues to have population statistics. Also, note that in October, 2022 VDW is 
+		still in the process of updating their CENSUS_DEMOG table and content.
+		This shortened version of CENSUS_DEMOG is required in CODI datamart because it is the 
+		target of several other CODI table foreign keys.
+	2. Changing field name CHORDS_GEOLEVEL to GEOLEVEL to match latest VDW CENSUS_DEMOG table. 
+	
+*/
+
+
 CREATE SCHEMA VDW;
 
 CREATE TABLE VDW.CENSUS_DEMOG (
   CENSUS_YEAR int NOT NULL,
   GEOCODE varchar(15) NOT NULL,
-  BLOCK varchar(3) NULL,
```

Comment:
These are valid columns according to the [CODI Data Models IG v4.1.5 (See Table 12 on page 46)](https://raw.githubusercontent.com/mitre/codi/main/CODI%20Data%20Model%20Implementation%20Guide.pdf)

Reply:
Should CENSUS_DEMOG just be totally removed? Looks like the draft CODI 4.2 DM IG removes this table.

Reply:
Yeah it's been commented out now but we do of course have git history so should probably delete it instead